tests/phpunit/integration/Core/Key_Metrics/REST_Key_Metrics_ControllerTest.php

```diff
@@ -187,6 +197,56 @@ public function test_setup_completed( $expected, $settings ) {
 		$this->assertEquals( $expected, $this->key_metrics_setup_completed->get() );
 	}
 
+	public function test_setup_completed_by_user_id() {
+		$this->key_metrics->register();
+		$this->settings->register();
+		$this->controller->register();
+		$this->register_rest_routes();
+
+		$initial_user = get_current_user_id();
+		$data         = apply_filters( 'googlesitekit_inline_base_data', array() );
```

Comment:
The Controller under test here isn't concerned with the inline base data here, so while that happens to be related due to its binding elsewhere, it isn't relevant in the context of testing the behavior of this class.

Instead, we should only be concerned with observing an expected value in the `Key_Metrics_Setup_Completed` value.